extensions/amp-fx-flying-carpet/0.1/amp-fx-flying-carpet.js

```diff
@@ -167,20 +166,50 @@ export class AmpFlyingCarpet extends AMP.BaseElement {
       this.element,
       this.children_
     );
-    listen(this.element, AmpEvents.BUILT, this.layoutBuiltChild_.bind(this));
+    this.observeNewChildren_();
```

Comment:
Did you mean to remove these changes? It seems to be the same changes as https://github.com/ampproject/amphtml/pull/23774.

I'm skipping this file assuming that is true.

Reply:
Yeah, just forgot to the update